docs-src/guide-rpc-connect.scrbl

```diff
@@ -0,0 +1,519 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag   "guide-rpc-connect"
+  ]{How to connect your language to the Reach RPC server}
+
+It's very easy to create a new a @tech{RPC} client library if one isn't already
+available for your platform. This section describes the necessary steps.
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@section[#:tag "rpc-connect-overview"]{Overview}
+
+As you know from the @seclink["guide-rpc-server-api"]{previous section},
+@tech{RPC} client @tech{frontends} communicate via two channels, named
+@pyin{rpc} and @pyin{rpc_callbacks}, which are obtained by calling
+@pyin{mk_rpc}. Construct these three functions and you'll have already
```

Comment:
I think this section is where you're talk about the ways that these clients are "typically" written, as opposed to the other sections where you'll talk about what the API *is* and then what the Python/Go client /provides/